mod-source-record-storage-server/src/main/java/org/folio/services/util/AdditionalFieldsUtil.java

```diff
@@ -587,10 +599,12 @@ && isNotBlank(record.getParsedRecord().getContent().toString())) {
         return parsedRecordContentCache.get(content);
       } catch (Exception e) {
         LOGGER.warn("computeMarcRecord:: Error during the transformation to marc record", e);
-        MarcReader reader = buildMarcReader(record);
-        if (reader.hasNext()) {
-          return reader.next();
-        }
+        try {
+          MarcReader reader = buildMarcReader(record);
+          if (reader.hasNext()) {
+            return reader.next();
+          }
+        } catch (Exception ignored) {}
```

Comment:
why doesn't it make sense to log the exception?

Reply:
done, thanks